tests/integration/plots/conftest.py

```diff
@@ -79,3 +81,109 @@ def make():
         yield image_v2, linear_v2, confusion_v2, confusion_props
 
     return make
+
+
+@pytest.fixture
+def repo_with_config_plots(tmp_dir, scm, dvc, run_copy_metrics):
```

Comment:
@pared I think we are missing tests for the case `3. Multiple data series` in https://github.com/iterative/dvc/pull/7477#issuecomment-1165429358

Reply:
Yes, its true. I will fix that in follow up PR, where I intend to parameterized this test to handle one use case per test. That way we won't have to create this huge `repo_with_plots`.
This specific use case (list of columns) is tested in `tests/unit/render/test_vega_converter.py::test_converter` - its a unit test but should be enough for now.